src/main/java/de/siegmar/logbackgelf/GelfTcpAppender.java

```diff
@@ -66,6 +67,13 @@
      */
     private int poolMaxWaitTime = DEFAULT_POOL_MAX_WAIT_TIME;
 
+    /**
+     * Maximum amount of time (in seconds) that a pooled connection can be idle before it is
+     * considered 'stale' and will not be reused. A value of -1 disables the max idle time feature.
+     * Default: -1 (disabled).
+     */
+    private int poolMaxIdleTime = DEFAULT_POOL_MAX_IDLE_TIME;
```

Comment:
Please also extend the documentation in the README.md file and the example file in the examples/ directory.